pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -10,7 +10,6 @@ ActivityInstanceStatusType:
     - COMPLETE
     - CREATED
     - IN_PROGRESS
-    - TERMINATED
```

Comment:
This change is not noted in the PR's description- why is this needed at this time?

Reply:
This status is no longer in the API (it has been removed some time ago). Just cleaning things up.

Reply:
Let's try and keep unrelated changes to their own PR in the future. Even if it's not being used, there may be client logic & dependencies on this and it does constitute a breaking change.

Reply:
Good point. However, I truly believe that this was added in here haphazardly and wasn't cleaned up. The chances of clients depending on this value is UNLIKELY since the sever will not return such a value. I'll like to remove this. This has been noted in DDP-4471. Resolving comment.